### DISCUSSION TITLE
Force using Ipv6

### DISCUSSION THREAD

**Discussion by Havard20009** [XTLS/Xray-core#2721] [Q&A]
hello
can you help me 
i want using my ipv6 only for my v2ray client
what should i do ?

thank you

**Comment by us254** [comment]
Update your V2Ray client configuration to use the IPv6 address of your VPS. This typically involves modifying the "address" field in the client configuration file to the IPv6 address.

**Reply by Havard20009** [reply]
i did it but wont work any other solution ?

**Comment by King-Of-Knights** [comment]
I am not sure whether I got the point, maybe you need `UseIPv6` in your outbound?
try to add this in your config `outbound`:
```
{
    "tag":"IPv6_out",
    "protocol": "freedom",
    "settings": {
        "domainStrategy": "UseIPv6"}
}
```
